tests/assets/imagenet_txt_dataset_custom_labels/train.txt

```diff
@@ -0,0 +1 @@
+1.jpg 0
```

Comment:
We used subdirectories for subformats and extra examples in the test assets. Probably, for this format it should also be done this way. Look at coco and voc directories.

Reply:
Okay, rearranged the assets.